ra/ra.go

```diff
@@ -1249,13 +1255,19 @@ func (ra *RegistrationAuthorityImpl) issueCertificateOuter(
 			prometheus.Labels{"type": "issued"},
 		).Observe(float64(len(order.Names)))
 
-		ra.newCertCounter.Inc()
+		ra.newCertCounter.With(
+			prometheus.Labels{
+				"profileName": profileName,
+				"profileHash": fmt.Sprintf("%x", profileHash),
```

Comment:
FWIW, `hex.EncodeToString(profileHash)` should be significantly faster than `fmt.Sprintf("%x", profileHash)`. The reason I recommended using `%x` last time was because it was already part of a much larger string formatting operation; here it's more efficient (and more readable IMO) to use the purpose-built function.